board/freescale/mx6ullevk/mx6ullevk_siklu_pcb19x_se_environment.c

```diff
@@ -54,25 +54,48 @@ static sf_env_siklu_se_t* p_sf_env_siklu_se = &sf_env_siklu_se;
 
 static int syseeprom_access_init = 0;
 
-static const char siklu_default_environment_se[] = { //
-		"SE_product_name=8010FX" ";" //
-						"SE_mac=00:24:a4:00:de:ad" ";"//
-						"SE_board_serial=F123456789" ";"//
-						"SE_system_serial=0" ";"//
-						"SE_port_map=cd--" ";" "\0" //
-		};
+static const char siklu_default_8020_environment_se[] = {
+		"SE_product_name=8020F" ";"
+		"SE_mac=00:24:a4:00:de:ad" ";"
+		"SE_board_serial=F123456789" ";"
+		"SE_system_serial=0" ";"
+		"SE_port_map=cd--" ";" "\0"
+};
+
+static const char siklu_default_8010_environment_se[] = {
+		"SE_product_name=8010FX" ";"
+		"SE_mac=00:24:a4:00:de:ad" ";"
+		"SE_board_serial=F123456789" ";"
+		"SE_system_serial=0" ";"
+		"SE_port_map=cd--" ";" "\0"
```

Comment:
for both: 8020F and 8010, can you please also add the following parameters?
SE_1588=0
SE_MCU_TYPE=60
@moshellib FYI

Reply:
Ok